include/LightGBM/cuda/vector_cudahost.h

```diff
@@ -42,8 +42,8 @@ struct CHAllocator {
   T* allocate(std::size_t n) {
     T* ptr;
     if (n == 0) return NULL;
-    n = (n + kAlignedSize - 1) & -kAlignedSize;
-    #ifdef USE_CUDA
+    n = (n + kAlignedSize - 1) / kAlignedSize * kAlignedSize;
```

Comment:
Can't we reuse `SIZE_ALIGNED` here?
https://github.com/microsoft/LightGBM/blob/d517ba12f2e7862ac533908304dddbd770655d2b/include/LightGBM/meta.h#L80

Reply:
Resolved via f44b881